app/src/main/java/blockeq/com/stellarwallet/helpers/LocalStore.kt

```diff
@@ -38,6 +38,11 @@ class LocalStore(private val sharedPreferences: SharedPreferences, private val g
         get() = getBoolean(KEY_PIN_SETTINGS_SEND)
         set(showPinOnSend) = set(KEY_PIN_SETTINGS_SEND, showPinOnSend)
 
+    var isPassphraseUsed : Boolean
```

Comment:
👍 Nice! We can probably use this to know whether it has been applied and indicate to the user in create/recover process for better experience.